src/util/fileLocking.ts

```diff
@@ -11,11 +11,14 @@ import { Duration } from '@salesforce/kit';
 import { retryDecorator } from 'ts-retry-promise';
 import { SfError } from '../sfError';
 import { Logger } from '../logger/logger';
+import { Global } from '../global';
 import { lockOptions, lockRetryOptions } from './lockRetryOptions';
 
 type LockInitResponse = { writeAndUnlock: (data: string) => Promise<void>; unlock: () => Promise<void> };
 type LockInitSyncResponse = { writeAndUnlock: (data: string) => void; unlock: () => void };
 
+export const noop = (): void => {};
```

Comment:
hypothesis: the in-memory fs is fast enough to not have synchronous read/write; also we're single-org in the web.

we can have these use locks if that turns out to be wrong